### PR TITLE
[DOCFIX] Simplify the Hive Integration doc

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Running-Hive-with-Alluxio.md [hunk 1]
```diff
@@ -32,21 +32,23 @@ First of all, set `HIVE_AUX_JARS_PATH` either in shell or `conf/hive-env.sh`:
...
-and internal tables. These tables can be new tables that are being created or existing tables that
-are stored in HDFS. Alluxio can also be used as the default file system for Hive. In the following
-sections, we will describe how to use Hive with Alluxio for these use cases. Hive is running on Hadoop
-MapReduce in this documentation.
 
-## Create External Table from Files Located in Alluxio
+## Create Hive Tables on Alluxio
 
-Hive can create external tables from files stored on Alluxio. The setup is fairly straightforward
+There al different ways to integrate Hive with Alluxio, as storage for both
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: typo: "al"

File: docs/en/Running-Hive-with-Alluxio.md [hunk 2]
```diff
@@ -32,21 +32,23 @@ First of all, set `HIVE_AUX_JARS_PATH` either in shell or `conf/hive-env.sh`:
...
+
+### Create New Tables from Files in Alluxio
+
+Hive can create new tables from files stored on Alluxio. The setup is fairly straightforward
 and the change is also isolated from other Hive tables. An example use case is to store frequently
 used Hive tables in Alluxio for high throughput and low latency by serving these files from memory
 storage.
 
-### Hive CLI examples
+#### Hive CLI examples for New Intenral Table
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: typo: "Intenral"

File: docs/en/Running-Hive-with-Alluxio.md [hunk 3]
```diff
@@ -101,75 +90,70 @@ FIELDS TERMINATED BY '|'
 LOCATION 'alluxio://master_hostname:port/ml-100k';
 ```
 
-## Use Alluxio as the FileSystem for External tables currently in HDFS
+The difference is that Hive will manage the lifecycle of internal tables.
+When you drop an internal table, Hive drops the table metadata and data both.
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: "drops the table metadata and data both" -> "deletes both the table metadata and the data."

File: docs/en/Running-Hive-with-Alluxio.md [hunk 4]
```diff
@@ -101,75 +90,70 @@ FIELDS TERMINATED BY '|'
...
+FIELDS TERMINATED BY '|';
 
-```
-hive> alter table TABLE_NAME set location "alluxio://master_hostname:port/ml-100k";
+hive> LOAD DATA LOCAL INPATH '/path/to/ml-100k/u.user'
+OVERWRITE INTO TABLE u_user;
 ```
 
-Use the following HiveQL and check the "Location" attribute to verify whether the table location is set correctly:
+Then use the following HiveQL we can change the table data location from HDFS to Alluxio：
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: "... the following HiveQL statement will change the table data ..."

File: docs/en/Running-Hive-with-Alluxio.md (hunk 1 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: fixed

File: docs/en/Running-Hive-with-Alluxio.md (hunk 2 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: done

File: docs/en/Running-Hive-with-Alluxio.md (hunk 3 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: done

File: docs/en/Running-Hive-with-Alluxio.md (hunk 4 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: done

**Comment by apc999** [comment]
@gpang  PTAL

**Pull request merged**